exchange-v2/contracts/ExchangeBulkV2.sol

```diff
@@ -0,0 +1,215 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.7.6;
+pragma abicoder v2;
+
+import "./ExchangeV2Core.sol";
+import "./RaribleTransferManager.sol";
+import "@rarible/royalties/contracts/IRoyaltiesProvider.sol";
+import "@rarible/exchange-interfaces/contracts/IWyvernExchange.sol";
+
+contract ExchangeBulkV2 is ExchangeV2Core, RaribleTransferManager {
```

Comment:
why is it inherited from RTM and exchangeV2?

Reply:
**done**, delete inheritance